.github/workflows/schedule-stale.yml

```diff
@@ -0,0 +1,35 @@
+on:
+  schedule:
+    - cron: "0 16 * * *"
+name: Stale Bot workflow
+jobs:
+  build:
+    name: stale
+    runs-on: ubuntu-latest
+    steps:
+      - name: stale
+        id: stale
+        uses: actions/stale@v3
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          days-before-stale: 30
+          days-before-close: 30
+          operations-per-run: 10
+          stale-issue-message: |
```

Comment:
Thoughts on adding labels for visibility? 🤔 

```
        stale-issue-label: 'no-issue-activity'

Reply:
https://github.com/actions/stale/blob/main/action.yml#L30
The default is to add the label `Stale` happy to change ti to something else though